packages/oscal-react-library/src/components/OSCALParam.tsx

```diff
@@ -0,0 +1,267 @@
+import {
+  StyledHeaderTableCell,
+  StyledTableHead,
+  StyledTableRow,
+} from "./OSCALSystemImplementationTableStyles";
+
+interface OSCALParamUsageProps {
+  usage: string | undefined;
+}
+
+const OSCALParamUsage: React.FC<OSCALParamUsageProps> = ({ usage }) => {
+  if (!usage) {
+    return null;
+  }
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Usage</OSCALSectionHeader>
+      <OSCALMarkupMultiLine>{usage}</OSCALMarkupMultiLine>
+    </>
+  );
+};
+
+interface OSCALParamConstraintsProps {
+  constraints: Constraint[] | undefined;
+}
+
+const OSCALParamConstraints: React.FC<OSCALParamConstraintsProps> = ({ constraints }) => {
+  if (!constraints) {
+    return null;
+  }
+
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Constraints</OSCALSectionHeader>
+      {constraints?.map((constraint) => (
+        <>
+          <OSCALMarkupMultiLine>{constraint.description}</OSCALMarkupMultiLine>
+          <Table>
+            <StyledTableHead>
+              <TableRow>
+                <StyledHeaderTableCell>Expression</StyledHeaderTableCell>
+                <StyledHeaderTableCell>Remarks</StyledHeaderTableCell>
+              </TableRow>
+            </StyledTableHead>
+            <TableBody>
+              {constraint.tests?.map((test) => (
+                <StyledTableRow key={test.expression}>
+                  <TableCell>{test.expression}</TableCell>
+                  <TableCell>{test.remarks}</TableCell>
+                </StyledTableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </>
+      ))}
+    </>
+  );
+};
+
+interface OSCALParamGuidelinesProps {
+  guidelines: Guideline[] | undefined;
+}
+
+const OSCALParamGuidelines: React.FC<OSCALParamGuidelinesProps> = ({ guidelines }) => {
+  if (!guidelines) {
+    return null;
+  }
+
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Guidelines</OSCALSectionHeader>
+      {guidelines?.map((guideline) => (
+        <OSCALMarkupMultiLine key={guideline.prose}>{guideline.prose}</OSCALMarkupMultiLine>
+      ))}
+    </>
+  );
+};
+
+interface OSCALParamValuesProps {
+  values: string[] | undefined;
+}
+
+const OSCALParamValues: React.FC<OSCALParamValuesProps> = ({ values }) => {
+  if (!values) {
+    return null;
+  }
+
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Values</OSCALSectionHeader>
+      <Typography>{values?.join(", ")}</Typography>
+    </>
+  );
+};
+
+interface OSCALParamLinksProps {
+  links: OSCALLink[] | undefined;
+}
+
+const OSCALParamLinks: React.FC<OSCALParamLinksProps> = ({ links }) => {
+  if (!links) {
+    return null;
+  }
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Links</OSCALSectionHeader>
+      {links?.map((link) => (
+        <Link key={link.href} href={link.href}>
+          {link.text ?? link.href}
+        </Link>
+      ))}
+    </>
+  );
+};
+
+interface OSCALSelectProps {
```

Comment:
For [select](https://pages.nist.gov/OSCAL/reference/latest/catalog/json-reference/#/catalog/params/select), I thought it made the most sense to actually select things with the [MUI Select component](https://mui.com/material-ui/react-select/).

I realized that there is not really a way to do anything with that selection other than displaying it currently, but it seemed better than a table of what you could select.